src/storage/src/store_impl.rs

```diff
@@ -92,11 +92,14 @@ impl StateStoreImpl {
     ) -> StorageResult<Self> {
         let store = match s {
             hummock if hummock.starts_with("hummock+") => {
-                let object_store = Arc::new(
+                let remote_object_store = Arc::new(
```

Comment:
StateStore does not need to concern about `remote_object_store` and `local_object_store`. If you want to flush some data to local object store, it's work of `WriteBufferUploader`